configure/ingress-nginx/mandatory.yaml

```diff
@@ -1,63 +1,56 @@
+
 apiVersion: v1
 kind: Namespace
 metadata:
   name: ingress-nginx
   labels:
     app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/instance: ingress-nginx
```

Comment:
I believe this remains `part-of` according to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/.